set.mm

```diff
@@ -530363,6 +530380,20 @@ be tempted to generalize this (or a similar) theorem by weakening the
       JUKULUMUN $.
   $}
 
+  ${
+    $d x y z $.  $d y z ph $.
+    $( One direction of ~ eu6 needs fewer axioms.  (Contributed by Wolf Lammen,
+       2-Mar-2023.) $)
+    wl-eu6-1 $p |- ( E. y A. x ( ph <-> x = y ) -> E! x ph ) $=
+      ( vz weq wb wal wex wi wa weu 19.42v pm3.33 alanimi equvelv sylib pm4.71i
+      albiim biancomi exbii equequ2 imbi2d albidv anbi2d syl5bb pm5.32ri bitr4i
+      ax6evr biantru 3bitr4ri exdistrv bitri wl-lem2-exsb anim1i sylbi sylibr
+      eu3v ) ABCEZFBGZCHZABHZABDEZIZBGZDHZJZABKUTURAIZBGZCHZVEJZVFUTVHVDJZDHZCH
+      VJUSVLCUSCDEZJZDHUSVMDHZJVLUSUSVMDLVKVNDVKVKVMJVNVKVMVKURVBIZBGVMVGVCVPBU
+      RAVBMNCDBOPQVMUSVKUSVHAURIZBGZJVMVKUSVHVRAURBRSVMVRVDVHVMVQVCBVMURVBACDBU
+      AUBUCUDUEUFUGTVOUSDCUHUIUJTVHVDCDUKULVIVAVEABCUMUNUOABDUQUP $.
+  $}
```

Comment:
Worth moving to Main.
Rk: for this and eu6, wouldn't it be more consistent to write `x = y  <-> ph` ?

Reply:
Also: I guess you have the same theorem as exsb with the reverse implication and E* ?

Reply:
I'd rather leave ph <-> x = y for consistency with df-mo.
Moving wl-eu6-1 to main requires a rewrite, since eu6 and this theorem have 80% of the proof in common, so a lemma is called for.  I don't have the time to do that now.

Reply:
move done

Reply:
> Also: I guess you have the same theorem as exsb with the reverse implication and E* ?

The definition of E* contains everything necessary.  Nothing to prove.